uiowa/uiowa
# Preserved link message feature

Resolves #5626 - As a SiteNow editor, I have a way to toggle a message to appear at the top of old news articles so that visitors are made aware of old links.

# How to test

- Pull down branch, sync a site. 
- Edit an existing article, check the Preserve Links box now available under the body field (do we like it there?)
- Save and view article, the default "Links in this article are preserved for historical purposes, but the destination sources may have changed." message should appear.
- Create a new article, see that the preserve link box is not available. Save and then edit it, it should now appear.
- While editing as a webmaster or administrator, click the link (that should open in a new tab) to change the message that displays. Change the placeholder text, save, check that the article now displays your custom text. You can also go back and empty that field, save, and see that the default message will reappear on articles. 
- Masquerade as editor or publisher, you should no longer see the link to change the message.
- Check that turning off the checkbox removes the message
- Repeat this process for iisc.uiowa.edu and sppa.uiowa.edu
